### PR TITLE
pkg(simple-apps): remove all recommendations of Simple apps

### LINKED ISSUE #158: Removal of Simple-Mobile Tools ( Simple Keyboard and Simple Gallery ) from recommendation sections
**Your phone model**: 

Any Xiaomi, Samsung, Pixel device

**Packages documentation to update:** 

```
com.samsung.android.honeyboard
com.touchtype.swiftkey
com.miui.gallery
com.google.android.inputmethod.latin
```

## Documentation Change

**List**: `Google`|`Misc`|`OEM` (manufacturer)

**Removal**:

None

### Current description

> Simple (product) is a good FOSS, lightweight replacement for system default, https://f-droid.org/en/packages/(package-link)

- The current description contains recommending certain apps from 'Simple-Mobile-Tools'.
- But as of 2024, sadly the company behind this suite of applications has been bought by an Ad-company.

Here are some related articles [1](https://www.ghacks.net/2023/12/05/warning-simple-mobile-tools-sold-to-controversial-zipoapps-publisher/), [2](https://www.androidauthority.com/simple-mobile-tools-acquisition-3391041/), [3](https://vid.puffyan.us/watch?v=OGr4H7QLVRs). 

Which would mean it can be controversial to recommend apps from these developers.

### Proposed description

Either remove these recommendations, or replace them with better ones like ( if not already present ) ['Unexpected Keyboard'](https://f-droid.org/en/packages/juloo.keyboard2), ['LeafPic-Revived'](https://f-droid.org/en/packages/com.alienpants.leafpicrevived).

### PR DESCRIPTION
**Pull request by Frigyes06**
Fixes #158

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -15923,7 +15923,7 @@
...
     "list": "Misc",
-    "description": "Swiftkey Keyboard (https://play.google.com/store/apps/details?id=com.touchtype.swiftkey)\nKeyboard app by TouchType, a Microsoft subsidiary (https://en.wikipedia.org/wiki/SwiftKey)\n4 Trackers + 11 Permissions: https://reports.exodus-privacy.eu.org/en/reports/com.touchtype.swiftkey/latest/\nNOTE: default keyboard on some Nokia and Huawei phones. Make sure you have another keyboard app before disabling this.\nSimple Keyboard is a good FOSS replacement based on the AOSP keyboard: https://f-droid.org/en/packages/rkr.simplekeyboard.inputmethod/",
+    "description": "Swiftkey Keyboard (https://play.google.com/store/apps/details?id=com.touchtype.swiftkey)\nKeyboard app by TouchType, a Microsoft subsidiary (https://en.wikipedia.org/wiki/SwiftKey)\n4 Trackers + 11 Permissions: https://reports.exodus-privacy.eu.org/en/reports/com.touchtype.swiftkey/latest/\nNOTE: default keyboard on some Nokia and Huawei phones. Make sure you have another keyboard app before disabling this.\nOpenBoard is FOSS, based on the AOSP keyboard and have direct boot support:\nhttps://f-droid.org/packages/org.dslul.openboard.inputmethod.latin/",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:15926]:
```suggestion
    "description": "Swiftkey Keyboard (https://play.google.com/store/apps/details?id=com.touchtype.swiftkey)\nKeyboard app by TouchType, a Microsoft subsidiary (https://en.wikipedia.org/wiki/SwiftKey)\n4 Trackers + 11 Permissions: https://reports.exodus-privacy.eu.org/en/reports/com.touchtype.swiftkey/latest/\nNOTE: default keyboard on some Nokia and Huawei phones. Make sure you have another keyboard app before disabling this.\nOpenBoard is FOSS, based on the AOSP keyboard and has direct boot support:\nhttps://f-droid.org/packages/org.dslul.openboard.inputmethod.latin/",
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Review by yjhn — CHANGES_REQUESTED** [review]
Simple Keyboard (https://f-droid.org/packages/rkr.simplekeyboard.inputmethod/, https://github.com/rkkr/simple-keyboard) is unrelated to Simple Mobile Tools.

So its recommendations shouldn't be removed.

File: resources/assets/uad_lists.json (hunk 1 above)
Comment (adhirajsinghchauhan) [reply resources/assets/uad_lists.json:15926]: Commit suggestions not working is a known GitHub bug. I've done it manually in 6271e0068f8b772e98543cc64ef442c8e2c42192 for all occurrences.

**Comment by AnonymousWP** [comment]
> [yjhn](/yjhn)

But the name is the same, I believe. How would users know the difference? Both are in the Play Store. I've linked to the Google Play Store one in the descriptions. At the same time, I feel like we shouldn't be in the position to recommend applications. It's mostly subjective, so I'm not a fan of advertising apps in the description like "choose app X, because of reason Y and Z!".

**Comment by yjhn** [comment]
> At the same time, I feel like we shouldn't be in the position to recommend applications. It's mostly subjective, so I'm not a fan of advertising apps in the description like "choose app X, because of reason Y and Z!".

I mostly agree (although there is the issue of discoverability). But if we decide to not recommend specific apps, all app recommendations should be removed.

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -15923,7 +15923,7 @@
...
+    "description": "Swiftkey Keyboard (https://play.google.com/store/apps/details?id=com.touchtype.swiftkey)\nKeyboard app by TouchType, a Microsoft subsidiary (https://en.wikipedia.org/wiki/SwiftKey)\n4 Trackers + 11 Permissions: https://reports.exodus-privacy.eu.org/en/reports/com.touchtype.swiftkey/latest/\nNOTE: default keyboard on some Nokia and Huawei phones. Make sure you have another keyboard app before disabling this.\nSimple Keyboard (https://play.google.com/store/apps/details?id=rkr.simplekeyboard.inputmethod) and OpenBoard are FOSS, based on the AOSP keyboard and has direct boot support:\nhttps://f-droid.org/packages/org.dslul.openboard.inputmethod.latin/",
```
Comment (yjhn) [inline resources/assets/uad_lists.json:15926]: > and has direct boot support

Maybe it should say something like "can be used as lockscreen keyboard" (assuming I understood [direct boot mode](https://developer.android.com/privacy-and-security/direct-boot) correctly from a cursory glance)?

**Review by yjhn — COMMENTED** [review]
Still need to decide on whether to recommend specific alternative apps, but it can wait.

**Comment by AnonymousWP** [comment]
> > At the same time, I feel like we shouldn't be in the position to recommend applications. It's mostly subjective, so I'm not a fan of advertising apps in the description like "choose app X, because of reason Y and Z!".
> 
> I mostly agree (although there is the issue of discoverability). But if we decide to not recommend specific apps, all app recommendations should be removed.

Correct, but that's *a lot* of work probably. Haven't checked how many entries there are though.

> Still need to decide on whether to recommend specific alternative apps, but it can wait.

Internally we already decided to not recommend specific alternative apps. We'll not allow it for future PRs/commits, but removing is probably a lot of work. We'll see what we can do. Others are also free to file PRs of course.

File: resources/assets/uad_lists.json (hunk 2 above)
Comment (AnonymousWP) [reply resources/assets/uad_lists.json:15926]: No, sorry, because then I'll have to explain which apps also support direct boot mode.

**Review by AnonymousWP — APPROVED** [review]

**Pull request merged**